1. Mexiko Create filenames filename-mappings and infofiles.py

```diff
@@ -283,16 +283,16 @@ def create_infofiles(row, filenames_file, not_ok_file):
     
     if pd.notnull(row["Personnamn / fotograf"]):
         if "Apenes" in row["Personnamn / fotograf"]:
-            infotext +="|photographer       =  " + "{{creator:Sigvald_Linné}} / Ola Apenes\n"
+            infotext +="|photographer       = " + "{{creator:Sigvald_Linné}} / Ola Apenes\n"
```

Comment:
See fix ee1acc7cf7de437dcea76fb96f1fd035f0bacad0 that is my suggested solution. Also did you solution in previous commit 9ddd7377ac5b42a862b234dfb97310e70680a0d3